lib/Models/TimelineStack.ts

```diff
@@ -74,7 +76,7 @@ export default class TimelineStack {
     });
   }
 
-  destroy() {
+  deactivate() {
     if (this._disposeClockAutorun) {
```

Comment:
`TimelineStack` has a reaction that keeps the time of workbench items in sync with the timeline clock. Previously this reaction was created inside the constructor, so it kept running even when the `Timeline.jsx` component is unmounted/hidden. I have moved the reaction to a an `activate` method which gets called in the `componentDidMount` method of `Timeline.jsx` and created a `deactivate` method that gets called in unmount hook.